.dev/scripts/copyRuntime.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+source .env
```

Comment:
I (and some others) use [autoenv](https://github.com/Tarrasch/zsh-autoenv) to setup my projects' environment so I don't want to be forced into using another file to load environment variables.

I'd just check whether the environment variables your script needs are set without providing a source. If you insist on keeping the _.env_ file, only source it if it exists, pretty please.

```sh
# needs to be in connector root directory (needs a `cd CONNECTORPATH`)
[ -f .env ] && source .env
```